spec/dummy/app/assets/javascripts/preview.coffee

```diff
@@ -8,9 +8,6 @@ $ ->
 $(document).on 'click', '.docs_toggle_button', ->
   $(@).closest('.docs_item').find('.docs_code').toggle()
 
-$(document).on 'update.dtpicker', '#datetime_picker_pick_a_date_and_time', (_, ts) ->
```

Comment:
Next time, ask before removing? I added this specifically in a PR: https://github.com/dobtco/dvl-core/pull/181. The goal was to show developers what value would be passed to the server.